eng/nuget/Microsoft.NETCore.Runtime.ICU.Transport.pkgproj

```diff
@@ -61,10 +65,39 @@
          bundled in the git repo.
 
          To force a build to package using the in-tree files, pass the parameter. -->
-    <File Condition="'$(OverrideBundledDatFiles)' != 'true'" Include="$(RepoRoot)\eng\prebuilts\browser\*.dat" TargetPath="runtimes\browser-wasm$(ExtraPackageName)\native\lib\%(Filename).dat" SkipPackageFileCheck="true" />
+    <File Condition="'$(OverrideBundledDatFiles)' != 'true'" Include="$(RepoRoot)\eng\prebuilts\wasm\*.dat" TargetPath="runtimes\browser-wasm$(ExtraPackageName)\native\lib\%(Filename).dat" SkipPackageFileCheck="true" />
     <File Condition="'$(OverrideBundledDatFiles)' == 'true'" Include="$(RepoRoot)\artifacts\bin\icu-browser-wasm\*.dat" TargetPath="runtimes\browser-wasm$(ExtraPackageName)\native\lib\%(Filename).dat" SkipPackageFileCheck="true" />
   </ItemGroup>
 
+  <!-- Wasi -->
+  <ItemGroup Condition="'$(AddAllTargetOSToPackage)' == 'true' or '$(PackageRID)' == 'wasi-wasm'">
+    <File Include="$(RepoRoot)\artifacts\bin\icu-wasi-wasm\emscripten-version.txt" TargetPath="runtimes\wasi-wasm\%(Filename)%(Extension)" SkipPackageFileCheck="true" />
```

Comment:
emscripten-version.txt doesn't make sense for wasi. I expect the file isn't even there for a wasi build

Reply:
yes shouldn't be there, removed.